pkg/noderesourcetopology/least_numa.go

```diff
@@ -37,6 +37,7 @@ const (
 )
 
 func leastNUMAContainerScopeScore(pod *v1.Pod, zones topologyv1alpha2.ZoneList) (int64, *framework.Status) {
+	lh := logging.Log()
```

Comment:
Is there a specific reason why we should call the logger at the begging of each function? or its just used as a shortcut?

Reply:
it's a transitional change. In this phase I'm doing the minimal change to move from global logger to `logr.Logger` instances. Most of these usages will be removed in the upcoming changes.